version.md

```diff
@@ -0,0 +1,94 @@
+# Versioning endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents a single version, matching a single item.
+
+Whether a version is accessible depends on `versioning.item.history.view.admin` configuration.  
+If this is set to true, the version can only be retrieved if the user is an admin of the related item
+
+## Create version
+
+TODO
+
+## Get single version
+
+**GET /api/versioning/version/<:versionId>**
+
+Provide version information for the version id.
+
+```json
+{
+  "id": "102",
+  "version": "2",
+  "type": "version",
+  "created": "2015-11-03T09:44:46.617",
+  "summary": "Fixing some typos in the abstract",
+  "_links": {
+    "versionhistory": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistory/10"
+    },
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101"
+    },
+    "eperson": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/eperson"
+    },
+    "item": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/item"
+    },
+    "workspaceitem": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/workspaceitem"
+    },
+    "workflowitem": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/workflowitem"
+    }
+  }
+}
+```
+
+Status codes:
+* 200 OK - if the version exists and is accessible by the current user
+* 401 Unauthorized - if you are not authenticated and versioning is not public
+* 403 Forbidden - if you are not logged in with sufficient permissions and versioning is not public
+* 404 Not found - if the version doesn't exist
+
+## Get single version for item
+
+**GET /api/versioning/item/<:item-uuid>**
+
+Provide version information for the item uuid.
```

Comment:
This endpoint initially looked odd to me, but I think the description here just wasn't making sense.  We might want to reword this to say something like: "Provide version information based on a given Item UUID. An Item UUID will only match one version."

Reply:
It's indeed a much clearer wording, I've updated it as such

Reply:
why not use the version id? if we need a search by item we can have a search method, this would be more inline with the other endpoint.
BTW: I don't think that this will provide a real used shortcut on the client side to move from the item to the version as this is not the read direction that is used in any use case that I can think about, instead as it is a different approach compared to other endpoint it could result confusing. I would also prefer `/api/versioning/versionitems` as endpoint (please note the final s)

Reply:
please look to https://github.com/DSpace/Rest7Contract/pull/97#discussion_r376520331it only need to be structured as a search method